atlasdb-impl-shared/src/test/java/com/palantir/atlasdb/sweep/queue/SweepQueueReadWriteTest.java

```diff
@@ -90,18 +91,19 @@ public static int putTombstone(KvsSweepQueueWriter writer, long timestamp, Cell
 
     private static int write(KvsSweepQueueWriter writer,
             long timestamp, Cell cell, boolean isTombstone, boolean conservative) {
-        WriteInfo write = WriteInfo.of(conservative ? TABLE_REF : TABLE_REF2, cell, isTombstone, timestamp);
+        TableReference tableRef = conservative ? TABLE_REF : TABLE_REF2;
+        WriteInfo write = WriteInfo.of(WriteReference.of(tableRef, cell, isTombstone), timestamp);
         writer.enqueue(ImmutableList.of(write));
         return write.toShard(SHARDS);
     }
 
-    public static List<WriteInfo> writeToUniqueCellsInSameShard(KvsSweepQueueWriter writer, long timestamp, long number,
+    public static List<WriteInfo> writeToUniqueCellsInSameShard(KvsSweepQueueWriter writer, long timestamp, int number,
             boolean conservative) {
         TableReference tableRef = conservative ? TABLE_REF : TABLE_REF2;
         List<WriteInfo> result = new ArrayList<>();
         for (long i = 0; i < number; i++) {
             Cell cell = getCellWithFixedHash(i);
-            result.add(WriteInfo.of(tableRef, cell, true, timestamp));
+            result.add(WriteInfo.write(tableRef, cell, timestamp));
```

Comment:
This should be `WriteInfo.tombstone`

Reply:
This was wrong before and got fixed when refactoring 🥇